apps/site-app/src/containers/App/index.tsx

```diff
@@ -37,30 +37,33 @@ export const App = () => {
   return (
     <MediaContextProvider>
       <style>{mediaStyles}</style>
-      <div id={styles.headerDecoration}>
+      <div id={styles.app}>
+
         <MainNavigation />
-      </div>
 
-      <div className={`${styles.contentContainer}`}>
-        <Container style={{ width: '100%' }} className={``}>
-          <MainPageTransition>
-            <GreaterThanMobileSegment>
-              <Rail internal position='left'>
-                <Sticky context={contextRef}>
-                  <PageSidebar />
-                </Sticky>
-              </Rail>
-            </GreaterThanMobileSegment>
 
-            <Ref innerRef={contextRef}>
-              <section id={styles.mainContent} className={styles.pageMainInner}>
-                <Routes />
-              </section>
-            </Ref>
-          </MainPageTransition>
-        </Container>
+        <div className={styles.menuPanel}>
+          <GreaterThanMobileSegment>
+            <Rail internal position='left'>
+              <Sticky context={contextRef}>
+                <PageSidebar />
+              </Sticky>
+            </Rail>
+          </GreaterThanMobileSegment>
+        </div>
+
+        <div className={styles.mainContent}>
+          <Ref innerRef={contextRef}>
+            <MainPageTransition routes={routes} contentId={styles.mainContent}>
+            </MainPageTransition>
+          </Ref>
+        </div>
+
+        <div className={styles.widgets}>
+          TODO Extra Widges
```

Comment:
TODO Extra Widges?

Reply:
This is actually supposed to be on the next PR, it was accidentally pushed here.